src/main/java/me/srgantmoomoo/postman/backend/util/Discord.java

```diff
@@ -18,7 +18,7 @@ public static void startRPC() {
         
         if(Minecraft.getMinecraft().player != null && Minecraft.getMinecraft().world != null) {
         	discordRichPresence.details = Minecraft.getMinecraft().player.getName();
-        }else discordRichPresence.details = "in da menus";
+        } else discordRichPresence.details = "in da menus";
```

Comment:
I like my elses with no spaces bfore them :). A lot of the code is this project looks weird with weird whitespace cause of how my eclipse was configured but my else’s are not mistakes.